src/features/import-projects/rpgf/index.ts

```diff
@@ -0,0 +1,15 @@
+import { getDataSource } from "../../../helpers/db";
+import { updateOrCreateProject } from "../helpers";
+import { rpgf3SourceConfig } from "./constants";
+import { fetchRpgf3Projects } from "./service";
+
+export const fetchAndProcessRpgf3Projects = async () => {
+  const data = await fetchRpgf3Projects();
+  if (!data) return;
+  const dataSource = await getDataSource();
+  if (!dataSource) return;
```

Comment:
I am uneasy with this! if any issue happens in getting data source, we pass silently. It's worth in fetching & processing Giveth projects, there we fetch projects in advance and if there is an issue with data source fetching we pass and fetch the next batch!
First, I think we can move that `getDataSource` calling into `updateOrCreateProject` method and don't pass it every time. Also wrapp it with good exception checking and logging in case any issue exists.